insights/parsers/tomcat_virtual_dir_context.py

```diff
@@ -0,0 +1,46 @@
+"""
+TomcatVirtualDirContext - command ``/bin/grep -R --include '*.xml' 'VirtualDirContext' /usr/share/tomcat*``
+===========================================================================================================
+
+This parser reads the output of ``/bin/grep -R --include '*.xml' 'VirtualDirContext' /usr/share/tomcat*``
+and detects whether there are any config files which contain VirtualDirContext.
+
+Sample input::
+
+    /usr/share/tomcat/conf/server.xml:    <Resources className="org.apache.naming.resources.VirtualDirContext"
+
+Examples::
+
+    >>> shared[TomcatVirtualDirContext].has_virtual_dir_context
+    True
+    >>> shared[TomcatVirtualDirContext].lines
+    ('/usr/share/tomcat/conf/server.xml',
+    '    <Resources className="org.apache.naming.resources.VirtualDirContext"')
```

Comment:
Do you keep the whitespace on any purpose? If not, better `strip` them.

Reply:
Yes, I want to show the exact offending line to the user.